dump-logs/README.md

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Set up and run some tests
       - name: Dump logs
         if: failure()
-        uses: canonical/charming-actions/dump-logs
+        uses: canonical/charming-actions/dump-logs@2.6.3
```

Comment:
Should we encourage pinning in the example?
```suggestion
        uses: canonical/charming-actions/dump-logs@main
```

Reply:
I think we should encourage pinning but probably not to this specific version. WDYT? Should I change this to 
```suggestion
        uses: canonical/charming-actions/dump-logs@<specific-version>
```